src/Functions/base32Decode.cpp

```diff
@@ -16,19 +16,30 @@ using FunctionBase32Decode = FunctionBaseXXConversion<Base32DecodeImpl>;
 
 REGISTER_FUNCTION(Base32Decode)
 {
-    factory.registerFunction<FunctionBase32Decode>(FunctionDocumentation{
-        .description = R"(
-Decode a [Base32](https://datatracker.ietf.org/doc/html/rfc4648) encoded string. The input string must be a valid Base32 encoded string, otherwise an exception will be thrown.)",
-        .arguments = {
-            {"arg", "A Base32 (rfc4648) encoded string"},
-        },
-        .examples = {
-            {"simple_decoding1", "SELECT base32Decode('ME======')", "a"},
-            {"simple_decoding2", "SELECT base32Decode('JBSWY3DP')", "Hello"},
-            {"empty_string", "SELECT base32Decode('')", ""},
-            {"non_ascii", "SELECT hex(base32Decode('4W2HIXV4'))", "E5B4745EBC"},
-        },
-        .introduced_in = {25, 5},
-        .category = FunctionDocumentation::Category::String});
+    FunctionDocumentation::Description description = R"(
+Decodes a string from [Base32](https://datatracker.ietf.org/doc/html/rfc4648#section-6), according to RFC 4648.
+Throws an exception in case of an error.
+)";
+    FunctionDocumentation::Syntax syntax = "base32Decode(encoded)";
+    FunctionDocumentation::Arguments arguments = {
+        {"encoded", "String column or constant. If the string is not valid Base32-encoded, an exception is thrown.", {"String"}}
+    };
+    FunctionDocumentation::ReturnedValue returned_value = {"Returns a string containing the decoded value of the argument.", {"String"}};
+    FunctionDocumentation::Examples examples = {
+    {
+        "Usage example",
+        "SELECT base32Decode('IVXGG33EMVSA====');",
+        R"(
+┌─base32Decode('IVXGG33EMVSA====')─┐
+│ Encoded                          │
+└──────────────────────────────────┘
+        )"
+    }
+    };
+    FunctionDocumentation::IntroducedIn introduced_in = {18, 16};
```

Comment:
So base32Decode was introduced in v18.16, but base32Encode was introduced in 25.6? That seems unlikely, maybe double-check?